src/cli-client/internals/cli/cli_upload.go

```diff
@@ -0,0 +1,150 @@
+package cli
+
+import (
+	"fmt"
+	"regexp"
+	"slices"
+	"strings"
+
+	"github.com/canonical/oci-factory/src/cli-client/internals/client"
+	"github.com/canonical/oci-factory/src/cli-client/internals/trigger"
+)
+
+type UploadRelease struct {
+	Track     string
+	Risks     []string
+	EndOfLife string
+}
+
+type CmdUpload struct {
+	UploadRelease []string `long:"release" description:"Release images to container registries.\nSyntax: --release track=<release track>,risks=<risk1>[,<risk2>...],eol=yyyy-mm-dd"`
+}
+
+var riskOptions = []string{"stable", "candidate", "beta", "edge"}
+
+func init() {
+	parser.Command.AddCommand("upload", "Trigger the build and release for the image in the current working directory",
```

Comment:
Why not use `parser.Command` directly?

Reply:
What do you mean by using `parser.Command` directly?

Reply:
Sorry `parser.AddCommand`. I must have been dozy!

Reply:
Nice one! Fixed in [c541687](https://github.com/canonical/oci-factory/pull/198/commits/c5416879a41a4c8447573cb9b9ef207bc73bffc9).